docs/_static/vcr/vcr_v2.yaml

```diff
@@ -416,7 +416,39 @@ paths:
                 $ref: "#/components/schemas/VerifiablePresentation"
         default:
           $ref: '../common/error_response.yaml'
+  /internal/vcr/v2/holder/{did}/vc:
+    post:
+      summary: Load a VerifiableCredential into the holders wallet.
+      description: |
+        If a VerifiableCredential is obtained in another way or when it's created without publishing, this API allows to add it to a wallet.
+        The DID of the holder has to be provided in the path.
+        It's assumed that the credentialSubject.id equals the holder DID.
```

Comment:
what if it doesn't? Do we still load it?

Reply:
Right now, yes. I think the internal wallet API needs to change. Instead of calling `Wallet()` on the VCR, it needs to be `Wallet(holderDID)` because not all VC.credentialSubject.ID needs to equal the holder DID theoretically.

Reply:
or add a wallet DID to the funcs (`Put(ctx, walletDID, credential)`). Our dependency injection style makes it cumbersome to have non-multi-tenant interfaces (see OpenID handlers)